### DISCUSSION TITLE
persistQueryClient & react native

### DISCUSSION THREAD

**Discussion by mightym** [TanStack/query#2547] [Q&A] [answered]
Hey guys!
I tried using persistQueryClient with expo and AsyncStorage https://react-native-async-storage.github.io/async-storage/docs/usage/ but no success so far. I wonder if someone could tell me if it is in general possible to use another storage  than the browser local storage? 

Thanks in advance!

**Comment by TkDodo** [comment, marked as answer]
Have you seen: https://react-query.tanstack.com/plugins/createAsyncStoragePersistor

**Reply by mightym** [reply]
Oh wow, no. Must have been blind :) I give it a try. Thanks a lot @TkDodo !

**Comment by iamit-code** [comment]
Hi, I am facing similar issues, can someone give the current URL ?

**Reply by TkDodo** [reply]
https://tanstack.com/query/v5/docs/framework/react/plugins/persistQueryClient
